### ISSUE TITLE
Linux desktop file refers to "Incognito" windows

### ISSUE THREAD

**Issue by rillian** [brave/brave-browser#37623]
## Description 

Our `brave-browser.deskop` file, included with the official Linux packages, refers to `brave --incognito` as "New Incognito Window". This should use our "New Private Window" terminology.

## Brave version (brave://version info)

1.64.122

## Version/Channel Information:

- Can you reproduce this issue with the current release?  Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Miscellaneous Information:

Afaict gnome desktops don't use this information, but it's used by several of the older launchers.

**Comment by fmarier** [comment]
Indeed, we'd have to add a patch for https://source.chromium.org/chromium/chromium/src/+/main:chrome/installer/linux/common/desktop.template;l=172;drc=a3ab59c808b58d126b6ca5fd588516c203ffa436 in our [existing patch](https://github.com/brave/brave-core/blob/b0827816c23c815ad6b7be6a67b17ad9f946e834/patches/chrome-installer-linux-common-desktop.template.patch).

**Comment by Rakib-mbstu** [comment]
Can I work on the issue?

**Comment by Pushkal-G** [comment]
Hey @fmarier , This is pretty handy browser , going through the repo. Pretty interesting, eager to work on something.Should I start working on this, or if there are some issues you can suggest, thats good too.

**Comment by MartiPresa** [comment]
Hi, is this issue still available?

**Comment by ShreyasSN** [comment]
Hi, I would like to work on this. Is this issue still available?

**Comment by AbhinavHaridas** [comment]
Can I work on this issue?

**Comment by anandude** [comment]
is this issue still available?

**Comment by softsideof** [comment]
Hi, I’d like to work on this issue and will open a PR shortly.

**Comment by softsideof** [comment]
Opened a fix here: https://github.com/brave/brave-core/pull/35781
